tpm2/constants.go

```diff
@@ -634,6 +634,18 @@ const (
 	TPMHTAC            TPMHT = 0x90
 )
 
+// Saved Context transient object handles.
+// See definition in Part 2: Structures, section 14.6.2
+// Context Handle Values come from table 211
+const (
+	// an ordinary transient object
+	TPMIDHSSavedTransient TPMIDHSaved = 0x80000000
```

Comment:
nit: These symbols don't have a name in the spec. I like your suggestions except for the extra S: could you name them `TPMIDHSavedTransient`, `TPMIDHSavedSequence`, etc.

Reply:
Updated in https://github.com/google/go-tpm/pull/351/commits/f3f60777432027152671c9a5dad0934124ace348